### ISSUE TITLE
Monero. Solo mining. Zeros in the results row.

### ISSUE THREAD

**Issue by RunesReader** [xmrig/xmrig#2415]
Hello!
I set up solo mining for Monero coins.
Is it ok, when I see **zeros** in the Results row **_all time_**?

Thank you in advance!
![Screenshot 2021-05-30 at 15 49 40](https://user-images.githubusercontent.com/12240504/120105030-fddb6580-c15f-11eb-812d-f6fbb623059e.png)

**Comment by SChernykh** [comment]
When solo mining, each accepted result = mined block, so it will be rare. You can check on testnet if it's working correctly, testnet difficulty is much lower.

**Comment by IgorArsenkinGL** [comment]
@SChernykh Thank you for quick response! I will try testnet.

**Comment by RunesReader** [comment]
@SChernykh Thank you for quick response! I will try testnet.

**Comment by toynn** [comment]
Hi - you should join a mining pool for Monero. There are a number of options like miningpoolhub.com and others. These payout based on your hashrate contribution not on how many blocks you actually solve. Solo mining with a low hashrate will take a very long time to be lucky to solve a block. Based on your hashrate and the total network hashrate you would expect to solve a block once every 6 years.

**Comment by RunesReader** [comment]
@toynn I was mining Monero in the pool. I had about 0.01 Monero per month. So it cannot be interesting at all. But solo mining is is like gambling :))

**Comment by Lonnegan** [comment]
> @toynn I was mining Monero in the pool. I had about 0.01 Monero per month. So it might not be interesting at all. But solo mining is is like gambling :))

Yes, but with such a low hashrate you have, it's a hopeless gamble. Using a calculator tool, you can see, that with just 1,6 kH/s the probability to find a block for yourself is at 6.5 years! Ok, probability doesn't mean, that it will take so long. You could find one tomorrow; or just in 20 years.

Like @toynn said, I'd mine on a PPS pool, where you get paid for each share you return to the pool, regardless if it found a block in that peroid or not.

**Comment by RunesReader** [comment]
@Lonnegan Yes, you are right. But for now I use my free laptop for mining only as a start point in mining world. I am just trying to understand all these stuffs. Thank you for response!
